<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="stdOutBox">
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="plainText">
       <string notr="true"/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="stdInBox">
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string notr="true"/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="inputLine">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>350</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="lineEndingSelector"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
